src/spawn_sync.cc

```diff
@@ -814,9 +814,7 @@ int SyncProcessRunner::ParseStdioOptions(Local<Value> js_value) {
   HandleScope scope(env()->isolate());
   Local<Array> js_stdio_options;
 
-  if (!js_value->IsArray())
-    return UV_EINVAL;
-
+  CHECK(js_value->IsArray());
```

Comment:
Are these changes related? Why is crashing better here? I think this one might be semver-major if it’s kept this way (or are there type checks somewhere else? In that case, could `ParseStdioOptions` be made more specific so it takes a `Local<Array>` instead?)